### ISSUE TITLE
Alternate or supplemental SIC usage

### ISSUE THREAD

**Issue by JimCircadian** [icenet-ai/icenet-project#50]
Likely to trial this with higher resolution SIC sources but applies to other potential sources of ground truth. Consider a pretraining phase at lower resolution stepping up to higher.

* https://github.com/icenet-ai/icenet/issues/161
  * Mask validations
* [ ] Trial [training pipeline](https://github.com/icenet-ai/icenet-pipeline) with alternate SIC truth
  * [ ] Straight through training
  * [ ] Low resolution pre-training workflow
* [ ] Trial [prediction pipeline](https://github.com/icenet-ai/icenet-pipeline) with alternate SIC

**Comment by JimCircadian** [comment]
AMSR 3km is implemented from the University of Hamburg. 6km can come from Bremen (they only supply 3km in HDF so easier just to use the alternative.) 

There's reason to extract and improve the downloaders for this issue, so reviewing the development effort to abstract out the common missing dates functionality and start addressing #51 as well as this.

**Comment by JimCircadian** [comment]
@MartinSJRogers  has posted this. Passive microwave data at 3.125km2 going back to 1979. Although this has data as brightness temperatures so would need to be converted to SIC via the algorithm: https://nsidc.org/data/nsidc-0630/versions/1
